sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java

```diff
@@ -77,6 +77,7 @@ public void describe(SensorDescriptor descriptor) {
 
   @Override
   public void execute(SensorContext context) {
+    System.out.println("EXECUTE DA SENSOR");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)